mlflow/store/tracking/dbmodels/models.py

```diff
@@ -198,6 +198,7 @@ def to_mlflow_entity(self):
         run_info = RunInfo(
             run_uuid=self.run_uuid,
             run_id=self.run_uuid,
+            run_name=self.name,
```

Comment:
And here https://github.com/mlflow/mlflow/pull/6782/files#r974607012 for `run_name` instead of `name`?

Reply:
Will update throughout to be as consistent as possible. In this case `name` was a pre-existing field so I tried to minimize diff, but can update that one too if you like.

Reply:
It seems like it makes for the least amount of confusion (since experiments have names too) :)

Reply:
Reverting this one change because it required a schema change which feels excessive for this PR. Everything else is standardized to `run_name`